cves/kernel/CVE-2022-1671.yml

```diff
@@ -84,14 +89,8 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: ff8376ade4f668130385839cef586a0990f8ef87
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+  note: "Manually confirmed"
```

Comment:
Is there anything notable about the fix?

Reply:
I believe we just have to leave these as "Manually confirmed" per the note placeholder for commits that were scraped.